visual-tests/src/test-fixtures/series/stacked-bar.js

```diff
@@ -32,7 +32,11 @@
             });
         });
 
-        series.crosshair = [];
+        var stackLayout = d3.layout.stack()
+            .x(function(d) { return d.state; })
+            .y(function(d) { return d.value; })
+        
+        var stackedData = stackLayout(series.map(function(d) { return d.data; }));
```

Comment:
Investigate the use of [d3.nest](https://github.com/mbostock/d3/wiki/Arrays#nest_entries), happy to see this done in a separate PR though.